src/app/ecosystem-explorer/[slug]/page.tsx

```diff
@@ -42,7 +42,11 @@ export function generateMetadata({ params }: EcosystemProjectProps) {
   const data = getEcosystemProjectData(slug)
 
   return createMetadata({
-    seo: { ...data.seo, image: graphicsData.ecosystem.data.src },
+    seo: {
+      ...data.seo,
+      title: `${data.title} - Ecosystem Project`,
```

Comment:
[Per David's message](https://filecoinfoundation.slack.com/archives/C03S74YDWMC/p1729901387615679)

```suggestion
      title: data.seo.title || `${data.title} - Filecoin Ecosystem Project`,
```